python-test/features/steps/local_agent.py

```diff
@@ -15,11 +16,20 @@ def run_local_agent_container(context):
     agent_docker_image = configs.get('agent_docker_image', 'ns1labs/orb-agent')
     image_tag = ':' + configs.get('agent_docker_tag', 'latest')
     agent_image = agent_docker_image + image_tag
-    env_vars = {"ORB_CLOUD_ADDRESS": orb_address,
-                "ORB_CLOUD_MQTT_ID": context.agent['id'],
-                "ORB_CLOUD_MQTT_CHANNEL_ID": context.agent['channel_id'],
-                "ORB_CLOUD_MQTT_KEY": context.agent['key'],
-                "PKTVISOR_PCAP_IFACE_DEFAULT": interface}
+    if "localhost" in base_orb_url:
+        env_vars = {"ORB_CLOUD_ADDRESS": 'localhost',
+                    "ORB_CLOUD_MQTT_ID": context.agent['id'],
+                    "ORB_CLOUD_MQTT_CHANNEL_ID": context.agent['channel_id'],
+                    "ORB_CLOUD_MQTT_KEY": context.agent['key'],
+                    "PKTVISOR_PCAP_IFACE_DEFAULT": interface,
+                    "ORB_TLS_VERIFY": "false"}
+    else:
```

Comment:
As far as I can tell, this `if` block is only different than the `else` block because of the `ORB_TLS_VERIFY` field.
In that case, let's add a config parameter for TLS verification rather than branching the code.
Question: why aren't HTTP calls also affected by this self-signed SSL certificates?